portal/server/routes/contract/eligibility/index.js

```diff
@@ -101,13 +109,18 @@ router.get('/contract/:_id/eligibility/supporting-documentation', [validateRole(
 
   const documentationValidationErrors = generateErrorSummary(supportingInformation.validationErrors, eligibilityErrorHref);
 
+  const criteriaValidationErrors = generateErrorSummary(eligibility.validationErrors, eligibilityErrorHref);
+
   const completedForms = completedEligibilityForms(deal.eligibility.status, validationErrors);
 
   return res.render('eligibility/eligibility-supporting-documentation.njk', {
     _id: deal._id,
     supportingInformation,
     eligibility,
-    validationErrors: documentationValidationErrors,
+    validationErrors: {
```

Comment:
Not for this PR, but I would be nice to only send `validationError` object if there is an error however looking at DTFS codebase this is not a widely followed practice.